tasks/EGPSR.tex

```diff
@@ -1,89 +1,76 @@
-Similar to a modern smart-speaker, the robot can be asked to do anything, in plain English, from either Stage of this and former rulebooks.
+The robot can be asked to do anything involving abilities required in \SONE{} and \STWO{} of this rulebook.
 
-% \subsection*{Focus}
-% This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
+\noindent \textbf{Focus:} \SysI{}.
 
 \subsection*{Main Goal}
-Execute at least one of the 3 commands requested by the operator.
+Execute 3 commands requested by the operator.
 
-\noindent\textbf{Reward:} 1500pts (500 points per task)\\
+\noindent\textbf{Reward:} 1500pts (500 points per command)
 
-\subsection*{Bonus rewards}
+\subsection*{Bonus Rewards}
 \begin{enumerate}[nosep]
-	\item Attend when called (125pts).
-	\item Understand a command given by naive operator (150pts, each).
-	\item Provide audio recording and transcript (100pts each).
-	\item Autonomously leaving the arena (150pts).
+	\item Understand a command given by naive operator (150pts, each)
+	\item Autonomously leaving the arena (100pts)
+	\item Complete all 3 commands in time with minimal help (200pts)
 \end{enumerate}
 
 % %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %
 % Setup
 %
 % %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection*{Setup}
+\subsection*{Setup and Procedure}
 \begin{enumerate}
-	\item \textbf{Location:} The task takes place inside the arena (some commands might require the robot to go out). The arena is in its normal state.
+	\item \textbf{Location:} The test mostly takes place inside the arena (some commands might require the robot to go outside).
+	
+	\item \textbf{Instruction Point:} An instruction point is assigned inside the arena where the robot has to go to receive a command.
 
-	\item \textbf{Start location:} The robot starts outside the arena. When the door opens, the robot moves towards the \textit{Standby Point}.
+	\item \textbf{Operators:} A \emph{professional} operator (i.e.~the referee) gives the commands. Optionally, teams can choose to receive commands by a \emph{naive} operator who has no background in robotics, i.e., an audience member. If the robot consistently fails to understand a command, the team can use a \CustomOperator{}
 
-	\item \textbf{Operators:} A \emph{professional operator} (i.e.~the referee) commands the robot by default.
+	\item \textbf{Operators:} A \emph{professional} operator (i.e.~the referee) gives the commands. Optionally, teams can choose to receive commands by a \emph{naive} operator who has no background in robotics, i.e., an audience member. The team can decide to use a \CustomOperator{} outright or after the robot fails to understand the professional or naive operator.
+	
+	\item \textbf{Minimal Help:} To score this bonus, the robot needs to complete the commands while requesting relatively few \abb{DEMs} compared to other teams. This is up to the consideration of the \abb{TC}.
```

Comment:
I also like the idea but it seems similar to just removing more points for each DeM and I suspect it will be quite annoying for the TC to handle on the day. If you want to go through with this, I suggest deciding how to handle it before hand, e.g:
"100 points for the team that used the least amount of DeM (several teams will score the bonus if the number of DeM is tied).